darwin/future/tests/core/items/test_archive_items.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import Mock
+from uuid import UUID
+
+import pytest
+import responses
+
+from darwin.exceptions import DarwinException
+from darwin.future.core.client import ClientCore
+from darwin.future.core.items.archive_items import archive_list_of_items
+from darwin.future.tests.core.fixtures import *
+
+
+@responses.activate
+def test_archive_items_including_filters(base_client: ClientCore) -> None:
+    # Define the expected response
+    responses.add(
+        responses.POST,
+        base_client.config.api_endpoint + "v2/teams/test-team/items/archive",
+        json={"affected_item_count": 0},
```

Comment:
Really minor point, but your affect item count here is mocked as 0, when if successful, it would've been 2.

Honestly, only fix if you want to.